exporter/awsemfexporter/metric_translator.go

```diff
@@ -159,10 +168,101 @@ func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, dimensionRollupOption stri
 				continue
 			}
 			cwMetrics := getCWMetrics(&metric, namespace, instrumentationLibName, dimensionRollupOption)
-			cwMetricList = append(cwMetricList, cwMetrics...)
+			cwMetricLists = append(cwMetricLists, cwMetrics...)
+			batchCWMetrics(cwMetrics, groupedCWMetricMap, cwMetricsMap)
 		}
 	}
-	return cwMetricList, totalDroppedMetrics
+	return cwMetricLists, totalDroppedMetrics
+}
+
+func batchCWMetrics(cwMetricList []*CWMetrics, groupedCWMetricMap map[string]*GroupedCWMetric, cwMetricsMap map[string]*CWMetrics) {
+	for _, met := range cwMetricList {
+		if len(met.Measurements[0].Dimensions[0]) > 0 {
+			key := strings.Join(met.Measurements[0].Dimensions[0], "")
+			if _, ok := cwMetricsMap[key]; ok {
+				for _, v := range met.Measurements[0].Metrics {
+					groupedCWMetricMap[key].Metrics[v["Name"]] = met.Fields[v["Name"]]
+					groupedCWMetricMap[key].MetricUnits[v["Name"]] = v["Unit"]
+				}
+			} else {
+				cwMetricsMap[key]=met
```

Comment:
nit: did you perform `go fmt` here?
```suggestion
				cwMetricsMap[key] = met
```